packages/frontend/src/Routes.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { lazy, Suspense } from "react";
-import { Router } from "@reach/router";
+import { BrowserRouter as  Router,Route ,Routes as ReactRoutes} from "react-router-dom";
```

Comment:
Looks like prettier is not run on this line
```suggestion
import { BrowserRouter as Router, Route, Routes as ReactRoutes} from "react-router-dom";
```

Can you run is explicitly, like it was done in other PRs?